src/applications/ivc-champva/10-10D/components/CustomSsnPattern.jsx

```diff
@@ -0,0 +1,34 @@
+import get from 'platform/utilities/data/get';
+import {
+  vaFileNumberUI,
+  ssnUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+// Using different hint text from the standard ssnOrVaFileNumberUI
+export const ssnOrVaFileNumberCustomUI = () => {
+  return {
+    ssn: ssnUI(),
+    vaFileNumber: {
+      ...vaFileNumberUI(),
+      'ui:options': {
+        hint:
+          'Enter this number only if it’s different than the Social Security number',
+      },
+    },
+    'ui:options': {
+      updateSchema: (formData, _schema, _uiSchema, index, path) => {
+        const { ssn, vaFileNumber } = get(path, formData) ?? {};
+
+        let required = ['ssn'];
+        if (!ssn && vaFileNumber) {
+          required = ['vaFileNumber'];
+        }
+
+        return {
+          ..._schema,
+          required,
+        };
+      },
+    },
+  };
+};
```

Comment:
Need to have the behavior of the stock ssn/vafilenumber, but with an extra hint over va file number.